samples/02_power_users_developers/introduction_to_hub_initiatives_and_sites.ipynb

```diff
@@ -0,0 +1,787 @@
+{
```

Comment:
<p>Cloning an initiative to Enterprise requires signing into that organization and cloning the site into it. You can clone a site by calling the clone method on the <code>sites</code> property of the target <code>GIS</code> object and providing a reference to the site object and an optional title.  </p><p>Note: Enterprise Sites are supported via the <code>sites</code> submodule of the <code>GIS</code> and not the <code>apps.hub</code> submodule. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1451/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>